src/js/store.jsx

```diff
@@ -17,7 +18,11 @@ const persistConfig = {
   blacklist: ['indicators', 'spinner', 'connection', 'infoBarVisibility', 'countWorkflow'],
 };
 
-const createStoreWithMiddleware = applyMiddleware(ReduxPromise, reduxThunk)(createStore);
+const createStoreWithMiddleware = applyMiddleware(
+  ReduxPromise,
+  reduxThunk,
+  cycleCount.validateCycleCount,
+)(createStore);
```

Comment:
so every time we add a middleware we'll need to remember to add it to this list?